python/ambassador/envoy/v2/v2listener.py

```diff
@@ -788,9 +791,8 @@ def __init__(self, config: 'V2Config', service_port: int) -> None:
         if self.config.ir.tracing:
             self.base_http_config["generate_request_id"] = True
 
-            self.base_http_config["tracing"] = {
-                "operation_name": "egress"
-            }
+            self.base_http_config["tracing"] = {}
+            self.traffic_direction = "OUTBOUND"
```

Comment:
This seems wrong-ish to me, but I think it's what preserves existing behavior.

Reply:
I think you're correct. @alexgervais might have some thoughts as to whether we should support configuring this.

Reply:
LGTM
At some point, `"operation_name": "egress"` was required for tracing to work on upstream calls. Having a quick look at the docs, and it would seem `traffic_direction` could be omitted entirely. Anyways, I'd be happy to test tracing manually before shipping.